### DISCUSSION TITLE
Office hour weekly

### DISCUSSION THREAD

**Discussion by jonesbusy** [jenkins-infra/plugin-modernizer-tool#1] [General]
Find a time slot for GSoC 2024 OpenRewrite Office Hours

**Comment by jonesbusy** [comment]
FYI @sridamul @berviantoleo @gounthar @iamrajiv

If none of the option, please suggest a time

**Reply by sridamul** [reply]
I am available at any time on UTC after 1:00pm on any day.

**Reply by jonesbusy** [reply]
Sure, let's wait for more votes and I will schedule recurring meeting starting from next week
